types/errors.go

```diff
@@ -0,0 +1,71 @@
+package types
+
+import "fmt"
+
+type IdentityError struct {
+	Code            int
+	Message         string
+	Internal        error
+	Did             string
+	ContentType     ContentType
+	DispalayMessage func() ResolutionResultI
```

Comment:
Why do we use lambda here?

Reply:
Can we just add `isDereferencing` property instead?

Reply:
Yes, thank you.
It seems that I tried to make the code as intimidating as possible with its seriousness. 😄 
Fixed